.circleci/config.yml

```diff
@@ -284,6 +284,8 @@ jobs:
   pre_integrationtest:
     macos:
       xcode: "11.2.1"
+    environment: 
+      TERM: xterm-256color
```

Comment:
not sure if we need this under `pre_integrationtest` since it's working without

Reply:
it's very odd, a recent `pre_Integrationtest` run is installing awscli 1.6.260, not 2.0.0

`==> Downloading https://homebrew.bintray.com/bottles/awscli-1.16.260.catalina.bo`

Reply:
added this in just in case it starts breaking in the future

Reply:
This flow uses the `s3` CLI command, vs. the `s3api` command. It's possible that it behaves differently, but it shouldn't hurt to explicitly specify the term setting here.